src/styles/menu.css

```diff
@@ -2,45 +2,62 @@
   overflow-y: auto;
   flex: auto;
   user-select: none;
-  /* transition: .2s ease-in; */
+  display: flex;
+  flex-direction: column;
+  align-items: center;
 }
 
 .sidebar--menu--header {
   cursor: default;
   text-align: center;
-  border-top: 2px solid black !important;
+  width: 100%;
+  padding: 15px 0px;
 }
 
 .sidebar--menu--header,
 .sidebar--menu--sub-header,
 .sidebar--menu--detail {
   cursor: pointer;
-  border: 1px solid black;
   outline: none;
 }
 
+.sidebar--menu--header,
+.sidebar--menu--sub-header {
+  font-size: 20px;
+}
+
+.sidebar--menu--sub-header {
+  align-items: center;
+  background: rgba(255,255,255,0.05);
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: space-between;
+  padding: 15px;
+  transition: all 300ms;
+}
+
 .sidebar--menu--sub-header:hover,
 .sidebar--menu--detail:hover {
-  background: rgba(39, 145, 152, 0.52) !important;
+  background: rgba(255,255,255,0.1);
 }
 
-.sidebar--menu:last-child {
-  border: 2px solid black;
+.sidebar--menu--detail.active {
+  background: rgba(255,255,255,0.05);
 }
```

Comment:
@IsaacAbrahamson added these styles here, which allows us to remove them individually from each of the themes and keeps us from repeating the same rules so often:
```csss
.sidebar--menu--sub-header {
  background: rgba(255,255,255,0.05);
}

.sidebar--menu--sub-header:hover,
.sidebar--menu--detail:hover {
  background: rgba(255,255,255,0.1);
}

.sidebar--menu--detail.active {
  background: rgba(255,255,255,0.05);
}
```

now we only need to add rules for the menu items when a theme calls for something different (like cobalt)